config/jobs/kubernetes/sig-network/ingress-gce-e2e.yaml

```diff
@@ -185,6 +186,13 @@ periodics:
       - --provider=gce
       - --test_args=--ginkgo.focus=\[Feature:Ingress\]|\[Feature:NEG\]
       - --timeout=320m
+      resources:
```

Comment:
Can we put some discussion of why these resources need to be added and where they are coming from in the commit message?

e.g. so git blame can be used to figure out why these exist and how they were obtained.

Reply:
Sure

Reply:
Updated.